### PR TITLE
Hotfix/tci builds

### PR DESCRIPTION
**Pull request by mfraezz**
Purpose
======
Fix build errors caused by recent refactoring.

Changes
=======
* Discontinue use of <code>csv</code> library.
* Use <code>pandas</code> for .tsv rendering
* Add test, test file for .tsv rendering
* Update image test path to new naming convention

Side Effects
=========
* .tsv rendering looks better than it did

### CODE REVIEW

File: mfr/ext/image/tests/test_image.py [hunk 1]
```diff
@@ -39,11 +39,11 @@ def test_render_img_tag(fakefile):
     assert 'alt="My image"' in result
 
 
-@pytest.mark.skipif(sys.version_info > (2, 7), reason="requires python2.7 or less")
+@pytest.mark.skipif(sys.version_info > (2, 8), reason="requires python2.7 or less")
```
Comment (sloria) [inline mfr/ext/image/tests/test_image.py]: Why was this changed?

Comment (mfraezz) [inline mfr/ext/image/tests/test_image.py]: It was being skipped in 2.7 tests, but the exporter should work in 2.6 and 2.7.
